week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -20,21 +20,37 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function() {
+  fillUp: function () {
+    this.volume = 100;
     // calling this function should pour your bottle full (volume = 100);
   },
   pour: function () {
+    if (this.volume < 100) {
+      return (this.volume += 10);
+    }

```

Comment:
This isn't quite right. If the volume were 95, for example, then adding 10 would increase the volume to 105.